Osiris/Hacks/AntiAim.cpp

```diff
@@ -6,23 +6,75 @@
 #include "../SDK/NetworkChannel.h"
 #include "../SDK/UserCmd.h"
 
+float AntiAim::RandomFloat(float min, float max) noexcept
+{
+    return (min + 1) + (((float)rand()) / (float)RAND_MAX) * (max - (min + 1));
+};
+
+void AntiAim::setPitch(float pitch, UserCmd* cmd, bool sendPacket) noexcept
+{
+    float newviewangle = pitch;
+
+    cmd->viewangles.x = newviewangle;
+};
+
+void AntiAim::setYaw(float yaw, UserCmd* cmd, bool sendPacket) noexcept
+{
+    float newviewangle = yaw;
+
+    if (config->antiAim.yawReal)
+    {
+        if (!sendPacket)
+        {
+            if (!GetAsyncKeyState(config->antiAim.yawInverseAngleKey))
+                newviewangle += localPlayer->getMaxDesyncAngle() * config->antiAim.bodyLean / 100;
+            else
+                newviewangle -= localPlayer->getMaxDesyncAngle() * config->antiAim.bodyLean / 100;
```

Comment:
this will only invert when u are holding the key

Reply:
yes I made it like that

Reply:
@3Ld4D bro i need your help in one rust server, contact me on Steam i added you, my name is El Cosito de La pizza i pay u for the help <3